### PR TITLE
Benp/add changelog

### PR DESCRIPTION
**Pull request by benpatterson**

### CODE REVIEW

**Comment by benpatterson** [comment]
@martenson @bramwelt this is in response to issues you logged; if you have time for a review, I'd appreciate your feedback. I did some lightweight grok'ing of the commit history to construct the info after 1.30.2

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+#### -> 1.32.3
+* Set the paramters again.
+
+#### -> 1.32.2
+* Remove adding parameters action from the env contributor; just add them all.
+
+#### -> 1.32.1
+* Allow for manual managing of webhooks
+* Various fixes, like:
+** Avoid overwriting test results when there's a race condition
```
Comment (martenson) [inline CHANGELOG.md]: this and few others below below have extra asterisks that render incorrectly

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+* Set the pull request after updating the updated timestamp. [See PR 368]
+
+#### -> 1.32.5
+* Pull new data in the case of a mergeability check
+
+#### -> 1.32.4
+* Don't check for comments when a new PR is opened.
+
+#### -> 1.32.3
+* Set the paramters again.
```
Comment (martenson) [inline CHANGELOG.md]: s/paramters/parameters

Comment (benpatterson) [inline CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (benpatterson) [inline CHANGELOG.md]: done

**Comment by benpatterson** [comment]
Thanks @martenson I addressed your comments and will go ahead and merge now.

We can go through a second round of updates if there's still confusion, but this gets us a relatively up-to-date changelog and that's cool.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+* Set the pull request after updating the updated timestamp. [See PR 368]
+
+#### -> 1.32.5
+* Pull new data in the case of a mergeability check
+
+#### -> 1.32.4
+* Don't check for comments when a new PR is opened.
+
+#### -> 1.32.3
+* Set the paramaters again.
```
Comment (martenson) [inline CHANGELOG.md]: This fix did not work out. 😆

Comment (benpatterson) [inline CHANGELOG.md]: haha. oh noes you're right!

I'll put in a quick PR to fix it (again!)

**Pull request merged**